_includes/home/team-gallery.html

```diff
@@ -0,0 +1,62 @@
+<section id="va-team-gallery">
+    <div class="grid-col-12 padding-0 margin-x-auto">
+        <div class="grid-container padding-3">
+
+            <div class="grid-row grid-gap">
+                <div class="grid-col-12 tablet:grid-col-12 tablet-lg:grid-col-12 text-center">
+                    <h3 class="gallery-heading">Meet the Speakers and Panelists</h3>
+                </div>
+                <hr class="gallery-divider" />
+            </div>
+
+            <!-- dynamic gallery tiles -->
+            <div class="grid-row grid-gap gallery--tile-container"></div>
+						
+        </div>
+    </div>
+</section>
+
+<!-- modal window -->
+<div class="gallery--modal-window">
+	<div
+		class="usa-modal"
+		id="gallery-modal"
+		aria-labelledby="modal-1-heading"
+		aria-describedby="modal-1-description"
+	>
+		<div class="usa-modal__heading text-right">
+			<button
+				type="button"
+				class="usa-button usa-button--unstyled padding-105"
+				data-close-modal
+			>
+				<img src="assets/img/icon.png" />
+			</button>
+		</div>
+
+		<div class="usa-modal__content">
+			<div class="usa-modal__main">
+				<div class="grid-row">
+
+					<div class="grid-col-4">
+						<img class="modal-img" src="" />
+					</div>
+
+					<div class="grid-col-8 right-column">
+						<h4 class="modal-name"></h4>
+						<p class="modal-department"></p>
+						<p class="modal-title"></p>
+						<p class="modal-organization"></p>
+					</div>
+				</div>
+				
+				<div class="grid-col-12 grid-row">
+					<p id="modal-1-description" class="modal-description"></p>
+				</div>
+
+			</div>
+		</div>
```

Comment:
Review on mobile. The alignment of image and text doesn't lend to a pleasant UX. Let's address the layout to appear more like the Card layout. Image centered, name/title below rather than to the right of the image.